ykrt/src/compile/jitc_yk/codegen/x86_64/mod.rs

```diff
@@ -341,8 +341,8 @@ impl<'a> X64CodeGen<'a> {
         // Operand types must be the same.
         debug_assert_eq!(lhs.type_(self.m), rhs.type_(self.m));
 
-        self.operand_into_reg(WR0, &lhs); // FIXME: assumes value will fit in a reg.
-        self.operand_into_reg(WR1, &rhs); // ^^^ same
+        self.load_operand(WR0, &lhs); // FIXME: assumes value will fit in a reg.
```

Comment:
I'm not 100% sure of this name, but let's push on and see how it feels.

(Does "load" imply usage of memory operand?)

Reply:
It's not perfect, but it matches some other existing functions, so I went with it.

Reply:
I think this stuff will sort itself out when Jake starts work on register allocation (when locals can live in either a register or a memory slot, rather than only in a memory slot)

Reply:
Agreed.